server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Properties.scala

```diff
@@ -25,9 +25,19 @@ import eu.timepit.refined.types.string.NonEmptyString
 import play.api.libs.json.JsObject
 
 object Properties {
+
+  val SPECIFIC_HEADER_PREFIX = "header:"
+
   def empty(): Properties = Properties()
 
   def apply(values: NonEmptyString*): Properties = Properties(values.toSet)
+
+  def sanitizeHeader(property: NonEmptyString): String = {
```

Comment:
This Email specific logic have nothing to do here IMO

Reply:
Why do we need to sanitize header properties?

Reply:
for insensitive case matching?